tests/programs/test_api_create_program.py

```diff
@@ -225,6 +224,29 @@ def test_api_dao_create_program__but_not_representative(self, mock_get_represent
         self.assertEqual(HTTPStatus.FORBIDDEN, response.status_code)
         self.assertEqual(messages.USER_IS_NOT_THE_ORGANIZATION_REPRESENTATIVE, response.json)
 
+    @patch("requests.get")
+    def test_api_dao_create_program__but_name_not_unique(self, mock_get_representative):
```

Comment:
can you please change the test name to "... _when_name_already_exist".

Reply:
Changes made